src/Components/BursaryItem.tsx

```diff
@@ -0,0 +1,23 @@
+//NPM packages
+
+//Project files
+import iBursaries from "interfaces/iBursaries";
+
+interface iProps {
```

Comment:
When dealing with things like testing, you would need to import export props. In those cases, it's better to be specific on what component the props belong to

```suggestion
interface BursaryProps {
```